app/locales/mr/exportscreen.json

```diff
@@ -0,0 +1,6 @@
+{
+    "export_para_1": "आपण खाली दिलेल शेयर बटण वापरून कुणाबरोबर ही आपली लोकेशन हिस्टरी शेयर करू शकता. बटण दाबले की आपल्याला लोकेशन हिस्टरी कुणाबरोबर आणि कशी शेयर करायची याबाबत सूचना मिळेल.",
+    "export_para_2": "लोकेशन हिस्टरी ही फक्त वेळ आणि स्थानिक निर्देशांकाच्या स्वरूपात शेयर केली जात आहे. तुमची कोणतीही वैयक्तिक / खाजगी माहिती शेयर केली जात नाही.",
```

Comment:
unless there is an absolute need to translate it this purely...sthanic nirdeshaank is very hard to understand. can we use the english word here?

Reply:
we are already using words like log cha data below... so maybe we can use the english equivalent of sthaanic nirdeshaank